.travis.yml

```diff
@@ -3,11 +3,11 @@ cache: bundler
 language: ruby
 
 rvm:
-  - 1.9.3
```

Comment:
A newer version of the `webmock` gem requires `addressable` that, in turn, requires a version of `public_suffix` that is no longer compatible with Ruby 1.9.x

Since I assumed I was working online on our fork, I removed the support because we don't need 1.9.x in Gaveteiro, but it's back on.